components/pipeline-service/staging/base/kustomization.yaml

```diff
@@ -15,6 +15,12 @@ resources:
   - ../../base/rbac
 
 patches:
+  - path: tekton-chains-signing-secret-name.yaml
+    target:
+      name: tekton-chains-signing-secret
```

Comment:
shouldn't we explicitly cite the namespace here?

Reply:
We don't do it [here](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/pipeline-service/staging/stone-stg-m01/resources/kustomization.yaml).

If you want to make it explicit, I would have no issues with it, but that does not seem necessary at the moment.

Reply:
ok sure if staging is OK let's keep it consistent

my question was from lack of certainty to be completely honest

feel free to resolve